fronts-client/src/components/feed/RecipeFeedItem.tsx

```diff
@@ -49,14 +53,19 @@ export const RecipeFeedItem = ({ recipe }: ComponentProps) => {
       type={CardTypesMap.RECIPE}
       id={recipe.canonicalArticle}
       title={recipe.title}
-      thumbnail={recipe.featuredImage.url}
+      thumbnail={recipe.previewImage?.url ?? recipe.featuredImage?.url ?? ""}
       liveUrl={`https://theguardian.com/${recipe.canonicalArticle}`}
       hasVideo={false}
       isLive={true} // We do not yet serve preview recipes
       handleDragStart={handleDragStart}
       onAddToClipboard={onAddToClipboard}
       shouldObscureFeed={shouldObscureFeed}
-      metaContent={<ContentInfo>Recipe</ContentInfo>}
+      metaContent={<>
+        <ContentInfo>Recipe</ContentInfo>
+        <ContentExtra>
+          {recipe?.score && recipe.score<1 ?  `Relevance ${Math.ceil(recipe.score*100)}%` : ""}
```

Comment:
Non-blocking, open question: do our users care about this? We're already seeing a sorted list of things. What does the user do with `Relevance: 82%`? (It feels useful for debugging – certainly something that might be useful to be switchable in `v2/features` for devs if we didn't include it on the main path)

Reply:
Yes this is mainly for us and for the Data Science people to understand where to place a relevancy cut-off and to better understand why certain things are coming up.
I also think it's more useful if we start putting in new sort-ordering